dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/filter/AggregateableDataElementFilter.java

```diff
@@ -33,15 +33,14 @@
 /**
  * @author Lars Helge Overland
  */
-public class AggregatableDataElementFilter
+public class AggregateableDataElementFilter
```

Comment:
We called it `Aggregatable` on purpose as that seems to be the accepted English word :-) 

https://en.wiktionary.org/wiki/aggregatable

https://www.yourdictionary.com/aggregatable

Reply:
Ok, I  found both versions, so I will change it to Aggregatable

Reply:
Great :-)